### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by CBenoit**

### CODE REVIEW

File: ironrdp/src/gcc/network_data/test.rs [hunk 1]
```diff
@@ -51,9 +51,7 @@ lazy_static! {
         io_channel: 1003,
         channel_ids: vec![1004, 1005, 1006],
     };
-    static ref CLIENT_NETOWORK_DATA_WITHOUT_CHANNELS: ClientNetworkData = ClientNetworkData {
```
Comment (thenextman) [inline ironrdp/src/gcc/network_data/test.rs:54]: typo "NETOWORK" > "NETWORK"

**Review by sduquette-devolutions — APPROVED** [review]
Approving the change to the dependabot config.

**Review by awakecoding — APPROVED** [review]

**Review by thenextman — APPROVED** [review]
Workflow LGTM

I only scanned through the code changes but it all seems like basic refactoring 👍

File: ironrdp/src/gcc/network_data/test.rs (hunk 1 above)
Comment (CBenoit) [reply ironrdp/src/gcc/network_data/test.rs:54]: Good catch!

**Comment by CBenoit** [comment]
Thank you!

Yes, it's only automated / basic changes + CI workflow :slightly_smiling_face:

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      matrix:
+        os: [ windows, linux ]
+        include:
+          - os: windows
+            runner: windows-2019
+          - os: linux
+            runner: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (dion-gionet) [inline .github/workflows/ci.yml:59]: Make sure to use checkout@v3 for every job

File: ironrdp/src/codecs/rfx/quantization.rs [hunk 3]
```diff
@@ -85,318 +84,262 @@ mod tests {
     }
 
     const QUANTIZIED_BUFFER: [i16; 4096] = [
```
Comment (thenextman) [inline ironrdp/src/codecs/rfx/quantization.rs:86]: Another one "QUANTIZIED" > "QUANTIZED"

**Review by dion-gionet — CHANGES_REQUESTED** [review]
Make sure to use

File: ironrdp/src/codecs/rfx/quantization.rs (hunk 3 above)
Comment (CBenoit) [reply ironrdp/src/codecs/rfx/quantization.rs:86]: Good catch!

**Comment by CBenoit** [comment]
> Make sure to use checkout@v3 for every job

Thank you! This is addressed.

**Review by dion-gionet — APPROVED** [review]

**Pull request merged**